src/context.tsx

```diff
@@ -1,8 +1,19 @@
 import React, { createContext, Dispatch, useReducer } from "react";
-import { AggregationFactor, Filters, FilterType, LanguageType, PageState, State } from "./types";
-import Translate from "./utils/translate/translateService";
+import {
+  AggregationFactor,
+  Filters,
+  FilterType,
+  LanguageType,
+  PageState,
+  State,
+} from "./types";
 
-export const AppContext = createContext({} as [State, Dispatch<Record<string, any>>]);
+export const AppContext = createContext(
+  {} as [State, Dispatch<Record<string, any>>]
+);
+
+const minDate = new Date(2019, 1, 1, 1);
```

Comment:
Lemme know if you guys don't like this approach - I had to do something like this because of the circular dependencies between `initialState` and `getEmptyFilters`.

Reply:
eh readable enough for now